lms/envs/aws.py

```diff
@@ -952,7 +953,7 @@
 # Enrollment URL used on the server-side.
 ENTERPRISE_ENROLLMENT_API_URL = ENV_TOKENS.get(
     'ENTERPRISE_ENROLLMENT_API_URL',
-    ENTERPRISE_ENROLLMENT_API_URL
+    (LMS_INTERNAL_ROOT_URL or '') + LMS_ENROLLMENT_API_PATH
```

Comment:
Should we default to `LMS_ROOT_URL` vs. `''`?

Reply:
`LMS_INTERNAL_ROOT_URL` [already defaults to `LMS_ROOT_URL`](https://github.com/open-craft/edx-platform/blob/a6b59342b684397df9de77f8d704d2ce275ee338/lms/envs/aws.py#L188).

I kept the `or ''` part because [this line](https://github.com/open-craft/edx-platform/blob/a6b59342b684397df9de77f8d704d2ce275ee338/lms/envs/aws.py#L950) makes it look like `LMS_ROOT_URL` can be `None` in some weird cases.

Reply:
Ah, ok :+1: